lock-api/src/main/java/com/palantir/lock/client/LockLeaseService.java

```diff
@@ -42,7 +42,7 @@
 import java.util.UUID;
 import java.util.stream.Collectors;
 
-class LockLeaseService {
+public class LockLeaseService {
```

Comment:
It seems off that you had to do this visibility change. I think it's probably an issue of changing where these are created (i.e. you want to create them inside factory methods - these are in many cases implementation details of the lock client)